rfd/0086/compat.md

```diff
@@ -0,0 +1,7 @@
+## Backwards compatibility and support
+
+The changes in this RFD are not backwards compatible with ContainerPilot 2.x configuration. For end users using Consul as their discovery backend, each container image can be safely updated independently; there is no communication between ContainerPilot instances except through the Consul API.
```

Comment:
I'd consider starting out by being explicit about the different interfaces that you're discussing here:

1. Interface with the discovery layer
2. Interface exposed to Prometheus-compatible clients (considered "beta," I believe)
3. Configuration file
3. Hooks and executables that ContainerPilot interacts with for health checks, etc.

Some of those interfaces are possibly unchanged, some are being changed, some changes are not backwards compatible with previous versions.